recipes/python-lmdb/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "python-lmdb" %}
+{% set pypi_name = "lmdb" %}
+{% set github_name = "py-lmdb" %}
+{% set version = "0.92" %}
+{% set sha256 = "e3eaf5f4a4d0fbae9c99bc803c21f91636c2c68dfd42ab1d58284ba4883695fb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Note: we're using the GitHub tarballs as the PyPI source distribution
+  # doesn't include the test suite.
+  fn: {{ github_name }}_{{ version }}.zip
+  url: https://github.com/dw/{{ github_name }}/archive/{{ github_name }}_{{ version }}.zip
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  imports:
+    - lmdb
+  commands:
+    - py.test {{ environ.get('SRC_DIR') }}/tests  # [not win]
```

Comment:
The linter is failing because of this line. You can use `$SRC_DIR` instead since you are already using `not win`.